src/cmd/root.go

```diff
@@ -143,6 +144,11 @@ func preRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	err := setUpConfiguration()
+	if err != nil {
```

Comment:
Umm... wouldn't it be more idiomatic to put this on the same line? We don't need `err` outside the error check.